node_modules/oae-core/resetpassword/js/resetpassword.js

```diff
@@ -16,144 +16,143 @@
+			'nospaces': oae.api.i18n.translate('__MSG__THE_USERNAME_SHOULDNT_CONTAIN_SPACES__')
+                    }
+		},
+		'methods': {
+                    'validchars': {
+			'method': function(value, element) {
+                            return this.optional(element) || !(/[<>\\\/{}\[\]!@#\$%\^&\*,:]+/i.test(value));
+			},
+			'text': oae.api.i18n.translate('__MSG__ACCOUNT_INVALIDCHAR__')
                     },
-                    //'text': oae.api.i18n.translate('__MSG__THIS_USERNAME_HAS_ALREADY_BEEN_TAKEN__')
-                    'text': "Username does not exist."
-                }
-            },
-            'submitHandler': getSecret
-        };
-        oae.api.util.validation().validate($('#resetpassword-input-username', $rootel), validateOpts);
-    };
+                    'usernameavailable': {
+			'method': function(value, element) {
+                            var response = false;
+                            isUserNameAvailable(value, function(available) {
+				response = available;
+                            });
+                            return response;
+			},
+			//'text': oae.api.i18n.translate('__MSG__THIS_USERNAME_HAS_ALREADY_BEEN_TAKEN__')
+			'text': "Username does not exist."
+                    }
+		},
+		'submitHandler': getSecret
+            };
+            oae.api.util.validation().validate($('#resetpassword-input-username', $rootel), validateOpts);
+	};
 		
-		/**
-		 * Get Secret: A start point for reseting passord
-		 */
-		var getSecret = function(form, event){
+	/**
+	 * Get Secret: A start point for reseting passord
+	 */
+	var getSecret = function(form, event) {
+            // Get the form values
+            var values = $('#resetpassword-input-username').serializeObject();
+            
+	    // disable redirect to other page
+	    event.preventDefault();
 
-          // Get the form values
-          var values = $('#resetpassword-input-username').serializeObject();
-          
-  				// disable redirect to other page
-  				event.preventDefault();
+	    var username = values.username;
 
-  				var username = values.username;
-          
-  				$.ajax({
-  						'url': '/api/auth/local/reset/init/'+username,
-  						'type': 'GET',
-  						'data': null,
-  						'success': function(data) {
-  				
-  							//callback(null, data);
-  							oae.api.util.notification('Congratulations!','A Email has been sent to your Email address.');
-  						},
-  						'error': function(data) {
-                
-  							oae.api.util.notification('Sorry!','An error occured while sending the Email!');
-  						}
-  		});
-		};
+	    $.ajax({
+  		'url': '/api/auth/local/reset/init/'+username,
+  		'type': 'GET',
+  		'data': null,
+  		'success': function(data) {
+  		    $('#resetpassword-modal').modal('hide');
+		    oae.api.util.notification('Congratulations!','A Email has been sent to your Email address.');
```

Comment:
Needs to be added to default.properties for translation. For the text, Email should be lowercase and the article should be 'An' instead of 'A'.